README.md

```diff
@@ -1 +1,36 @@
-# INF 6420 - Intro to Web Development
+# Emily Morris's Media Portfolio 
+
+A portfolio showcasing my writing work.
+
+## Wireframes 
+
+![wireframe of Landing Page](wireframes/IMG_1509.jpg)
```

Comment:
Hi @EmilyMorris10 please include images of your wireframes and not the images you will use on your website. You can see examples from your classmates: 
- https://github.com/INF6420/ss22-he8079
- https://github.com/INF6420/ss22-kimp7777
- https://github.com/INF6420/ss22-CanCui6420
- https://github.com/INF6420/ss22-CBreul
- https://github.com/INF6420/ss22-samiluke
- https://github.com/INF6420/ss22-Srhymes1010

Reply:
Thank you! With more time, I decided on a different topic too.